### DISCUSSION TITLE
How can I use cuda pytorch without zluda.exe

### DISCUSSION THREAD

**Discussion by KamioRinn** [lshqqytiger/ZLUDA#4] [Q&A]
I Install HIP SDK 5.7 and ZLUDA. 
Add their folders to PATH and replace HIP SDK library (6750XT gfx1031) from ROCmLibs.
Install torch by ```pip install torch==2.2.0 torchvision --index-url https://download.pytorch.org/whl/cu118```
Replace ```cublas64_11.dll cusparse64_11.dl nvrtc64_112_0.dll ```to ```torch\lib```.

Then I got False from:
```python
python
torch.cuda.is_available()
``` 

And got True from:
```python 
zluda python 
torch.cuda.is_available()
```

But I saw you use torch.cuda in other project without zluda.exe. How can I do there?
Thank.

**Comment by lshqqytiger** [comment]
If there are required dll files in PATH, `torch.cuda.is_available()` should return True.

**Comment by radna0** [comment]
I tried running ```zluda python``` but it does not seem to work, nothing happens in the terminal
